coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 99.64
+    "covered_percent": 0.0
```

Comment:
Right? Weird, huh?

Anyways, this just needs some test failure fixes and it'll be GTG